scripts/process_nf.py

```diff
@@ -207,7 +208,36 @@
                                               process_args=process_args,threshold=img_threshold,ome_dilation_iter=ome_dilation_iter,num_digits=6,stem=stem)
 
     
-    
+#==============================================================================
+# %% NEAR FIELD - splitting
+#==============================================================================
+
+max_RAM = 1  # in GB
```

Comment:
See above.  Not good to hardcode things when you can ask the system for available memory.  Available means not having to use disk.